### PR TITLE
More informative reducing test

### PR DESCRIPTION
**Pull request by dapi**

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#16](https://github.com/zold-io/zold-score/issues/16) is now in scope, role is `REV`

**Comment by emilianodellacasa** [comment]
@dapi Every commit message should start with the issue code. Also, please add the issue code to the PR title and please also add a meaningful description to this PR.

**Comment by emilianodellacasa** [comment]
@dapi ping

**Comment by 0crat** [comment]
@emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by dapi** [comment]
@emilianodellacasa thanks! created an issue - https://github.com/zold-io/zold-score/issues/23

File: test/zold/test_score.rb [hunk 1]
```diff
@@ -44,13 +44,22 @@ def around
...
+  # Reduces suffixes list
+  #
   def test_reduces_itself
     score = Zold::Score.new(
       time: Time.parse('2017-07-19T21:24:51Z'),
       host: 'localhost', port: 443, invoice: 'NOPREFIX@ffffffffffffffff',
       suffixes: %w[A B C D E F G]
-    ).reduced(2)
+    )
+
```
Comment (yegor256) [inline test/zold/test_score.rb]: @dapi don't use empty lines please: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html

Comment (dapi) [reply test/zold/test_score.rb]: empty lines are removed

**Comment by emilianodellacasa** [comment]
@dapi Add a description and the code of the originating issue

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (10 days), architects (@yegor256) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#16](https://github.com/zold-io/zold-score/issues/16) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@emilianodellacasa According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by emilianodellacasa** [comment]
@ypshenychka Confirmed

**Comment by ypshenychka** [comment]
@emilianodellacasa thanks

**Comment by ypshenychka** [comment]
@dapi ping

**Comment by ypshenychka** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request closed without merging**